templates/shared/pack.js

```diff
@@ -0,0 +1,20 @@
+const { readFileSync } = require('fs');
+const { parse, resolve } = require('path');
+const AdmZip = require('adm-zip');
+
+const { base } = parse(__dirname);
+const { version } = JSON.parse(
+  readFileSync(resolve(__dirname, 'build', 'manifest.json'), 'utf8')
+);
+
+const outdir = 'release';
+const filename = `${base}-v${version}.zip`;
+const zip = new AdmZip();
+zip.addLocalFolder('build');
+zip.writeZip(`${outdir}/${filename}`);
```

Comment:
I am facing an error on this line where it is unable to create `release` directory and works if `release` directory is already present.
```cmd
Error: ENOENT: no such file or directory, mkdir 'release/release'
...
errno: -2,
syscall: 'mkdir',
code: 'ENOENT',
path: 'release/release'
```